llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -52786,6 +52786,43 @@ static SDValue combineStore(SDNode *N, SelectionDAG &DAG,
     }
   }
 
+  // Convert store(cmov(load(x), y), x) to cstore(y, x).
```

Comment:
Maybe write out the CC operand in the comment?

Reply:
Done.